server/src/main/java/com/objectcomputing/checkins/services/questions/QuestionController.java

```diff
@@ -24,8 +35,26 @@
     @Inject
     private QuestionServices questionService;
 
-    public void setQuestionService(QuestionServices questionService) {
-        this.questionService = questionService;
+    public QuestionController(QuestionServices questionServices) {
+        this.questionService = questionServices;
+    }
+
+    @Error(exception = QuestionNotFoundException.class)
+    public HttpResponse<?> handleDupe(HttpRequest<?> request, QuestionNotFoundException e) {
```

Comment:
`handleDupe` on `QuestionNotFoundException`?

Reply:
maybe `handleDupe` just needs a rename?

Reply:
Looks like it, there are two handleDupes